tests/apptesting.py

```diff
@@ -117,14 +117,19 @@ def add(self, change: dict):
     def filter(self, since: str = "", limit: int = 500):
 
         first = 0
+        found = False
```

Comment:
@jamilatta, olhando agora o código percebi que essa função inteira poderia ser refatorada para a seguinte list-comprehension:

```python
    def filter(self, since: str = "", limit: int = 500):
        return [change
                for timestamp, change in self._data_store.items()
                if timestamp >= since][:limit]
```